consensus/istanbul/validator/weighted.go

```diff
@@ -383,6 +384,9 @@ func (valSet *weightedCouncil) SubListWithProposer(prevHash common.Hash, propose
 
 	// seed will be used to select a random committee
 	seed, err := ConvertHashToSeed(prevHash)
+	if fork.Rules(view.Sequence).IsIstanbul {
+		seed += view.Round.Int64()
+	}
```

Comment:
Not related to this change, however, we're maintaining almost identical code below.. (please check consensus/istanbul/validator/default.go )

Reply:
> It is still Reset to me.. Could you please check?

@ehnuje oh! I noticed that change is missing too.. i fixed it, sorry for confusion!

>  Not related to this change, however, we're maintaining almost identical code below.. (please check consensus/istanbul/validator/default.go )

Can I ask again what your question means..? Both changes are intended..